tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -148,6 +146,8 @@ public:
          FlushShadowWritePage();
       }
 
+      void *dst = fWritePage[fWritePageIdx].GrowUnchecked(count);
```

Comment:
Consider adding an explanation of why this should be here rather than somewhere else.
For example:
 ```suggestion
      // Given that RPage::GrowUnchecked() affects the number of elements returned 
      // by RPage::GetNElements(), the main write page should only be grown after the
      // check for flushing the shadow page. 
      void *dst = fWritePage[fWritePageIdx].GrowUnchecked(count);
```